adapter_test.js

```diff
@@ -37,22 +37,47 @@ test("post a job to queue", async () => {
   await a.delete("test");
 });
 
-
 test("get error jobs from the queue", async () => {
   // setup
   await a.create({
     name: "test2",
-    target: "https://jsonplaceholder.typicode.com/posts"
-  })
-  
+    target: "https://jsonplaceholder.typicode.com/posts",
+  });
+
   // test
-  const result = await a.get({name: 'test2', status: 'ERROR'})
-  assertEquals(result.ok, true)
+  const result = await a.get({ name: "test2", status: "ERROR" });
```

Comment:
All ought to be using an enum-like object or module for `status`

Reply:
I think it is ok to explicit in the test, but if you disagree I can create a const.